### DISCUSSION TITLE
Possibilities for ignoring non-required GitHub CI checks?

### DISCUSSION THREAD

**Discussion by andersfischernielsen** [renovatebot/renovate#23554] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36.18

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Renovate can be configured to ignore one or more CI/branch protection checks in GitHub either by workflow file name or by workflow name to allow auto-merging when the failing workflow isn’t marked as Required in the GitHub branch protections. 

### What you tried so far.

I looked through the issues and discussions here as well as the configuration documentation, and I couldn’t find a way to make Renovate ignore specific (non-required) CI checks in GitHub. 

The workflow I want to ignore is not marked as Required in the repository branch protections and should therefore not prevent Renovate from auto-merging the dependency branch. 

Is it possible to explicitly make Renovate ignore the outcome of a specific GitHub workflow? 

### Relevant debug logs

<details><summary>Logs</summary>

```
N/A
```

</details>

**Comment by viceice** [comment]
no, use platform automerge

**Reply by andersfischernielsen** [reply]
What about CI checks running for commits to the dependency branches Renovate creates? They are automerged if they pass, but Renovate expects every single check to pass, not just the required checks (according to branch protections) 🤔

**Reply by viceice** [reply, marked as answer]
platform automerge uses the GitHub automerge, which bypasses non required status checks.

renovate can't know which checks are mandatory. the API is only accessable to admins.
